testing/python/issue/test_tilelang_issue_2568.py

```
@@ -0,0 +1,87 @@
+import tilelang.language as T
+import tilelang.testing
+
+
+M, N = 32, 32
+THREADS = 128
+
+
+def _make_reduce_kernel(dtype, reduce_fn):
+    @T.prim_func
+    def main(A: T.Tensor((M, N), dtype), Out: T.Tensor((M,), dtype)):
+        with T.Kernel(1, 1, threads=THREADS) as (bx, by):
```

Ruff (0.16.1)

[warning] Unpacked variable `bx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)